src/index.html

```diff
@@ -21,7 +24,482 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a
+        class="header__picture"
+        href="./images/logo.png"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO-logo"
+        />
```

Comment:
The href attribute in the anchor tag should link to a webpage, not an image. If you want to display an image, you should use the img tag, which you are already doing. So, you can remove the href attribute from the anchor tag.